wire-library/wire-gradle-plugin/src/main/kotlin/com/squareup/wire/gradle/WireInput.kt

```diff
@@ -80,27 +90,32 @@ internal class WireInput(var configuration: Provider<Configuration>) {
 
     if (converted is File) {
       val file = if (!converted.isAbsolute) File(project.projectDir, converted.path) else converted
-      inputFiles.add(file)
+      if (!path.mayBeProject) inputFiles.add(file)
 
       return when {
         file.isDirectory -> project.dependencies.create(project.files(path))
         file.isJar -> project.dependencies.create(project.files(file.path))
+        path.mayBeProject -> {
+          // Keys can be either `path` or `configuration`.
+          // Example: "[path: ':someProj', configuration: 'someConf']"
+          return project.dependencies.project(mutableMapOf("path" to path))
```

Comment:
This would fail with something like
`Project with path ':bad-name' could not be found in project ':mobile:android:investing-protos'`
so I think it's fine if we don't try/catch it.